src/settingentry_gui.cpp

```diff
@@ -820,6 +820,7 @@ SettingsContainer &GetSettingsTree()
 			limitations->Add(new SettingEntry("construction.max_tunnel_length"));
 			limitations->Add(new SettingEntry("station.never_expire_airports"));
 			limitations->Add(new SettingEntry("vehicle.never_expire_vehicles"));
+			limitations->Add(new SettingEntry("vehicle.vehicle_intro_randomisation"));
```

Comment:
Is randomisation a "Limitation"?
I would put it under "Vehicles". (at the end, after the sub folders, to match "Competitors")